src/System.Net.HttpListener/src/System/Net/Unix/ChunkedInputStream.cs

```diff
@@ -0,0 +1,188 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
```

Comment:
I believe this second line shouldn't be here (same for the other files imported from mono).  @AlexGhiondea?